tests/workflows/basic_retry_node/workflow.py

```diff
@@ -20,15 +20,15 @@ def run(self) -> Outputs:
 class Subworkflow(BaseWorkflow[RetryNode.SubworkflowInputs, BaseState]):
     graph = StartNode
 
-    class Outputs(StartNode.Outputs):
-        pass
+    class Outputs(BaseWorkflow.Outputs):
+        execution_count = StartNode.Outputs.execution_count
```

Comment:
we have a ticket for workflows auto inferring outputs from their last nodes, which I think could help clean this up